sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/TelemetryItem.cs

```diff
@@ -13,15 +13,15 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Models
     internal partial class TelemetryItem
     {
         private const string DateTimeFormat = "yyyy-MM-ddTHH:mm:ss.fffffffZ";
-        private static readonly IReadOnlyDictionary<TelemetryType, string> s_partA_Name_Mapping = new Dictionary<TelemetryType, string>
+        private static readonly IReadOnlyDictionary<TelemetryType, string> s_telemetryItem_Name_Mapping = new Dictionary<TelemetryType, string>
```

Comment:
nit: We could remove this dictionary and directly map based in `activity.GetTelemetryType()` result. We could consider this in different PR too.

Reply:
sure - I will do this as follow up PR